cms/services.ts

```diff
@@ -18,18 +19,25 @@ interface Service {
   location?: string
 }
 
-export interface ServicePreview {
+export interface ServiceDetail {
```

Comment:
Maybe Pick from Service?

https://www.typescriptlang.org/docs/handbook/utility-types.html#picktype-keys

Reply:
This is really useful, thank you! `ServicePreview` and `ServiceDetail` are indeed both different subsets of `Service`, I've carded to make them pick in the next card that needs new keys.